receiver/awscontainerinsightreceiver/internal/host/ebsvolume.go

```diff
@@ -141,6 +142,12 @@ func (e *ebsVolume) addEBSVolumeMapping(zone *string, attachement *ec2.VolumeAtt
 func (e *ebsVolume) findNvmeBlockNameIfPresent(devName string) string {
 	// for nvme(ssd), there is a symlink from devName to nvme block name, i.e. /dev/xvda -> /dev/nvme0n1
 	// https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/nvme-ebs-volumes.html
+
+	// Windows does not support file system eg: /rootfs to get Nvme Block name.
+	// todo: Implement logic to identify Nvme devices on Windows. Refer https://docs.aws.amazon.com/AWSEC2/latest/WindowsGuide/nvme-ebs-volumes.html#identify-nvme-ebs-device
+	if runtime.GOOS == "windows" {
```

Comment:
Can we create "windows"  constant in constants util file and use it everywhere?

Reply:
Addressed